packages/cf-style-const/src/variables.js

```diff
@@ -60,5 +71,6 @@ export default {
   colorPurpleLight: lighten(colorOffsetLight, '#d91698'),
   colorPurpleDark: darken(colorOffsetDark, '#d91698'),
   colorTwitterBlue: '#00aced',
-  colorFacebookBlue: '#3b5998'
+  colorFacebookBlue: '#3b5998',
+  mainBackground: '#e6e6e6'
```

Comment:
Should this be `colorMainBackground` or `colorLightishGray`? What's our policy on exporting variables for their *purpose*, rather than for what they are?